TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/http/HttpJsonTransformingConsumer.java

```diff
@@ -52,7 +52,7 @@ public class HttpJsonTransformingConsumer implements IPacketFinalizingConsumer<A
     private final List<List<Integer>> chunkSizes;
     // This is here for recovery, in case anything goes wrong with a transformation & we want to
     // just dump it directly.  Notice that we're already storing all of the bytes until the response
-    // comes back so that we can format the output that goes to the comparator.  These should be
+    // comes back so that we can format the output that goes to EFS and eventually the comparator.  These should be
```

Comment:
EFS has nothing to do with this class.  Please strike.
"... output that goes downstream"